md/barostat/utils.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import networkx as nx
+from scipy.spatial.distance import pdist
+from typing import List
+from timemachine.lib.potentials import HarmonicBond
+
+
+def compute_box_volume(box: np.ndarray) -> float:
+    assert box.shape == (3, 3)
+    return np.linalg.det(box)
```

Comment:
Nice! I hadn't realized `det(box)` was the proper way to generalize volumes to triclinic systems, but after some searching it appears you are correct.

https://journals.iucr.org/a/issues/2011/01/00/au5114/au5114.pdf